include/ncasset/DefaultAssets.h

```diff
@@ -0,0 +1,74 @@
+/**
+ * @file DefaultAssets.h
+ * @copyright Jaremie Romer and McCallister Romer 2025
+ * 
+ * @note Assets in this file are guaranteed to be loaded before each call to
+ *       `Scene::Load()`.
+ */
+#pragma once
+
+#include "ncasset/AssetType.h"
+
+#include <span>
+#include <string_view>
+
+namespace nc::asset
+{
+/** @brief Name of the default plane mesh asset. */
+constexpr auto PlaneMesh = "default/plane.nca";
+
+/** @brief Name of the default cube mesh asset. */
+constexpr auto CubeMesh = "default/cube.nca";
+
+/** @brief Name of the default sphere mesh asset. */
+constexpr auto SphereMesh = "default/sphere.nca";
+
+/** @brief Name of the default capsule mesh asset. */
+constexpr auto CapsuleMesh = "default/capsule.nca";
+
+/** @brief Name of the default skybox mesh asset. */
+constexpr auto SkyboxMesh = "default/skybox.nca";
+
+/** @brief Name of the default wheel mesh asset. */
+constexpr auto WheelMesh = "default/wheel.nca";
+
+/** @brief Name of the default base color texture asset. */
+constexpr auto DefaultBaseColor = "default/color.nca";
+
+/** @brief Name of the default normal texture asset. */
+constexpr auto DefaultNormal = "default/normal.nca";
+
+/** @brief Name of the default roughness texture asset. */
+constexpr auto DefaultRoughness = "default/metallic.nca";
```

Comment:
We actually don't need DefaultRoughness and DefaultMetallic anymore until we do a different shader

Reply:
Totally delete both?